common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -53,11 +53,11 @@ case class TransactionId private (meta: TransactionMetadata) extends AnyVal {
    * @param message An additional message that is written into the log, together with the other information.
    * @param logLevel The Loglevel, the message should have. Default is <code>InfoLevel</code>.
    */
-  def mark(from: AnyRef, marker: LogMarkerToken, message: String = "", logLevel: LogLevel = InfoLevel)(
+  def mark(from: AnyRef, marker: LogMarkerToken, message: String = "", logLevel: LogLevel = DebugLevel)(
```

Comment:
i see in a few places below that you set the log level on the markers for to info level.
i hesitate with this global change - how many markers are affected and will now not log a message - can we avoid it?

Reply:
after looking through all the uses of `mark` this change is ok. 
here are the remaining ones for reference.

```
core/controller/src/main/scala/whisk/core/controller/Controller.scala:  TransactionId.controller.mark(
core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala:      transid.mark(
core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala:      transid.mark(
common/scala/src/main/scala/whisk/common/TransactionId.scala:  def mark(from: AnyRef, marker: LogMarkerToken, message: String = "", logLevel: LogLevel = DebugLevel)(
```

Reply:
@vvraskin can you set the ones in the invoker supervisor to info level also? these should be infrequent.

Reply:
But warn level should be also fine right? If info is enabled, warn messages should be printed as well

Reply:
Ah - you're right. Thanks.